CONTRIBUTING.md

```diff
@@ -1,27 +1,100 @@
-# Setup
-
-After you clone this repository run configure.ps1 & build.ps1.
-
-Note that build.ps1 is the only process producing all nuget packages and artifacts, building from visual studio is good for development purposes, but is not enough at the moment.
-
-### Build.ps1 is responsible for the following steps:
+# Contributing
 
-1. Update submodules
-1. Restore nuget packages
-1. Build
-1. ilmerge nuget.exe
-1. Running all unit tests
+## Prerequisites
 
-# Building in Visual Studio
+- [Visual Studio 2019](https://www.visualstudio.com)
+  with following workloads:
+  - .NET Core Cross Platform Development
+  - .NET desktop development
+  - Visual Studio extension development.
+  - Desktop development with C++
+- [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
+- Git
+- Windows Powershell v3.0+
 
-Follow the instructions in the README.
+> Note that you can work on the NuGet.Client repo with Visual Studio 2017, but you will be unable to test the Visual Studio extension.
```

Comment:
Is this still true now that we depend on MSBuild 16.4 ?

Reply:
Yes.